python/paratext/core.py

```diff
@@ -28,6 +28,10 @@
 
 import paratext_internal as pti
 
+import sys
+if sys.version_info < (3,):
+    range = xrange
```

Comment:
`six` may be able to help with this

https://pythonhosted.org/six/#module-six.moves